src/02-common-principles.md

```diff
@@ -37,15 +37,18 @@ misunderstanding we clarify them here.
     scanning sequence/protocol.
 
 1.  **Data type** - a functional group of different types of data.
-    BIDS defines eight data types:
-    `func` (task based and resting state functional MRI),
-    `dwi` (diffusion weighted imaging),
-    `fmap` (field inhomogeneity mapping data such as field maps),
-    `anat` (structural imaging such as T1, T2, PD, and so on),
-    `meg` (magnetoencephalography),
-    `eeg` (electroencephalography),
-    `ieeg` (intracranial electroencephalography),
-    `beh` (behavioral).
+    BIDS defines the following data types:
```

Comment:
I think we need `perf` here too.

Reply:
thanks! added below, please feel free to improve the wording with a suggestion.

Reply:
:pray: looks good to me